src/main/resources/view/ResultDisplay.fxml

```diff
@@ -3,6 +3,6 @@
 <?import javafx.scene.control.TextArea?>
 <?import javafx.scene.layout.StackPane?>
 
-<StackPane fx:id="placeHolder" styleClass="pane-with-border" xmlns="http://javafx.com/javafx/18" xmlns:fx="http://javafx.com/fxml/1" style="-fx-background-color: #333;">
-  <TextArea fx:id="resultDisplay" editable="false" maxHeight="170" minHeight="170" prefHeight="170" style="-fx-background-color: #333;" styleClass="result-display" />
+<StackPane fx:id="placeHolder" styleClass="pane-with-border" xmlns="http://javafx.com/javafx/18" xmlns:fx="http://javafx.com/fxml/1" style="-fx-background-color: #f1f2f6;">
```

Comment:
Not from your PR, but I think we should revert it back to javafx 8. Most likely due to SceneBuilder.
```suggestion
<StackPane fx:id="placeHolder" styleClass="pane-with-border" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1" style="-fx-background-color: #f1f2f6;">
```